code/style.css

```diff
@@ -1,3 +1,463 @@
+:root {
+  --primary: #F6E9D7;
+  --secondary: #D1ADCC;
+  --tertiary: #C2D2BD;
+  --accent: #5F2C3E;
+}
+
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin: 0;
+  background: var(--tertiary);
+  font-family: 'Manrope', sans-serif;
+}
+
+@media screen and (min-width:1024px) {
+
+  body {
+    display: flex;
+    flex-direction: row;
+  }
+}
+
+
+
+/**********************USER INFO SECTION*************************/
+
+.user-section {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  gap: 1rem;
+  margin-top: 1rem;
+}
+
+.user-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 0.5rem;
+  padding-bottom: 4rem;
+  border-block-end: 1px solid var(--secondary);
+  text-align: center;
+}
+
+.user-image {
+  width: 300px;
+  border-radius: 50%;
+  margin-bottom: 1rem;
+}
+
+@media screen and (min-width:768px) {
+
+  .user-section {
+    gap: 2rem;
+    margin: 2.5rem 0 0;
+  }
+
+  .user-info {
+    gap: 1.5rem;
+    padding-bottom: 6rem;
+  }
+
+  .user-image {
+    width: 350px;
+    margin-bottom: 0.5rem;
+  }
+}
+
+@media screen and (min-width:1024px) {
+
+  .user-section {
+    width: 30%;
+    max-height: 160vh;
+    justify-content: space-evenly;
+    gap: 0;
+    margin: 0;
+    padding: 0 1rem 0 3.5rem;
+  }
+
+  .user-info {
+    gap: 0;
+    margin: 0;
+    padding: 0;
+    border: none;
+    text-align: center;
+    align-items: center;
+  }
+
+  .user-image {
+    width: 225px;
+    margin: 0;
+    padding-top: 1rem;
+  }
+}
+
+@media screen and (min-width:1400px) {
+
+  .user-section {
+    width: 40%;
+    max-height: 130vh;
+    padding: 1rem 0 0 3.5rem;
+  }
+
+  .user-image {
+    width: 275px;
+    padding: 0;
+  }
+}
```

Comment:
I can see that you put all the media queries for each part that needs to be responsive underneath each other, you can put all the media queries that are the same for example all with the min-width 1024px in the same curly brackets for the entire page under each other to make it even easier to find where they are and it gives you a shorter code!

Reply:
This is actually an interesting topic! I've gathered the media queries as you described in the end of the css stylesheet in previous projects. This time I tried this different approach. In this way the extra code does come from repeating the media query headline several times (the content of course stays the same), but the benefit of the positioning like this is that you don't have to scroll down the page back and forth when setting media queries or changing something for a spesific element, this was way easier when developing the code. But I'm still not totally sure myself which approach I like best in general 😅  Here's an interesting discussion about this topic :https://css-tricks.com/forums/topic/where-to-put-css3-media-queries/

Reply:
Also, thank you Mimmi for my first 'proper' code review! 🥳  😄  🙏

Reply:
Yes, that is interesting! I'm not totally shore about with one  technigo "prefer" but I do think that whatever way you do it is most important to be consistent and do the same and that you did 🥳